I've yet to find any desktop workload that benefits from anything higher than
a swappiness as low as possible. Unfortunately setting it to zero can rarely
lead to unexplained stalls in the VM so use 10.

-ck

---
 mm/vmscan.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux-3.4-ck3/mm/vmscan.c
===================================================================
--- linux-3.4-ck3.orig/mm/vmscan.c	2012-06-01 21:37:23.000000000 +1000
+++ linux-3.4-ck3/mm/vmscan.c	2012-07-03 14:00:41.835312458 +1000
@@ -153,7 +153,7 @@ struct mem_cgroup_zone {
 /*
  * From 0 .. 100.  Higher means more swappy.
  */
-int vm_swappiness = 60;
+int vm_swappiness = 10;
 long vm_total_pages;	/* The total number of pages which the VM controls */
 
 static LIST_HEAD(shrinker_list);
